app/helpers/solidus_signifyd/name.rb

```diff
@@ -0,0 +1,17 @@
+module SolidusSignifyd
```

Comment:
```suggestion
# frozen_string_literal: true

module SolidusSignifyd
```

Reply:
Thanks! I may need to spend some time today to see why my linter is not picking these things up!

Reply:
This is not fixed yet, right?

Reply:
Updated with latest push

Reply:
Fixed in latest push